### PR TITLE
test(functional): fix post verify account recovery test

### PR DESCRIPTION
**Pull request by ashrivastava-qa**
## Because

- Fix pre verification step for post verify tests
- Disable monitor test until FXA-8446 is worked on

## This pull request

- Fixes pre verification step for post verify tests
- Disables monitor test until FXA-8446 is worked on

## Issue that this pull request solves

Closes: FXA-8281

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: packages/functional-tests/tests/postVerify/accountRecovery.spec.ts [hunk 1]
```diff
@@ -3,44 +3,27 @@
...
 import { test, expect } from '../../lib/fixtures/standard';
-const password = 'passwordzxcv';
-let email;
 
 test.describe('severity-1 #smoke', () => {
   test.describe('post verify - account recovery', () => {
     test.beforeEach(async ({ target, pages: { login } }) => {
       // Generating and consuming recovery keys is a slow process
       test.slow();
-      email = login.createEmail();
```
Comment (vbudhram) [inline packages/functional-tests/tests/postVerify/accountRecovery.spec.ts:14]: I wonder if there is a reason why we manually create a new account via the backend here.

**Review by vbudhram — APPROVED** [review]
@ashrivastava-qa LGTM 👍🏽

File: packages/functional-tests/tests/postVerify/accountRecovery.spec.ts (hunk 1 above)
Comment (ashrivastava-qa) [reply packages/functional-tests/tests/postVerify/accountRecovery.spec.ts:14]: AFAIR, this was kind of just a short-cut way to create account when you don't want it to be pre-verified and to go through the hassle of creating an account and go through the whole steps

**Comment by ashrivastava-qa** [comment]
Thanks @vbudhram for the review! :)

**Pull request merged**